### DISCUSSION TITLE
How to add dynamic  query parameter to the web hook url of directus???

### DISCUSSION THREAD

**Discussion by thakurPooja2494** [directus/directus#2690] [Archive]
How to add query parameter to the web hook url of directus and query parameter should be taken from the collection field..for which we have applied web hook
![image](https://user-images.githubusercontent.com/64476559/80595294-1d2a2f80-8a42-11ea-9202-7a4b3b612fb7.png)

**Comment by rijkvanzanten** [comment]
There is no such thing. Webhooks don't have access to the item that was changed / created.

Feel free to open a feature request for that! Sounds useful to support the `{{}}` syntax in that URL field

**Reply by thakurPooja2494** [reply]
@rijkvanzanten  can we add query param by creating custom hook.php file???

**Reply by rijkvanzanten** [reply]
Yeah! In a custom hook, you're free to do whatever. If you setup a hook using the [PHP Hooks](https://docs.directus.io/extensions/hooks.html#hooks) you'll be able to use the data that was saved

**Reply by thakurPooja2494** [reply]
can you help me to create a custom hook as when i have create a new file my-hook.php. directus is not opening or giving me 500 error and one more thing can you tell me where i can check the logs of custom hooks

**Reply by benhaynes** [reply]
Hey @thakurPooja2494 — apologies, but our team is very busy at the moment. Here are your options for support: https://docs.directus.io/getting-started/troubleshooting.html#technical-support
